internal/app/mistapiconnector/mistapiconnector_app.go

```diff
@@ -579,34 +584,55 @@ func (mc *mac) triggerUserNew(w http.ResponseWriter, r *http.Request, lines []st
 			req.Header.Add("Livepeer-Signature", signature_header)
 		}
 
-		glog.V(model.DEBUG).Infof("Calling playback.user.new webhook %s", userWebhook.Url)
-
 		resp, err := http.DefaultClient.Do(req)
+		webhookStatus := livepeer.WebhookStatus{
+			LastTriggeredAt: time.Now().Unix(),
```

Comment:
This will not output a timestamp compatible with the API. You need to do or have a helper like thiis:
https://github.com/livepeer/livepeer-data/blob/main/pkg/data/time.go#L16
to do a `UnixMillis`.